www/src/components/guidelines/nav-item.js

```diff
@@ -21,20 +23,27 @@ const NavLink = styled(ActiveLink, { shouldForwardProp })(
   space
 )
 
-NavLink.defaultProps = {
-  color: `navigation.linkDefault`,
-  fontFamily: `heading`,
-  fontSize: 3,
-  fontWeight: `body`,
-  mr: 4,
-}
-
 NavLink.propTypes = {
   ...propTypes.color,
   ...propTypes.space,
   ...propTypes.typography,
 }
 
-const NavItem = props => <NavLink {...props} />
+const NavItem = props => (
+  <NavLink
+    sx={{
+      // override the default link styling
+      "&&": {
+        border: 0,
+        color: `navigation.linkDefault`,
+        fontFamily: `header`,
```

Comment:
Breaking change in `gatsby-design-tokens` v2: `header` is now `heading`:

```suggestion
        fontFamily: `heading`,
```